release notes/v0.43.0.md

```diff
@@ -0,0 +1,124 @@
+- Native support for JavaScript's `async`/`await`.
+- A new experimental JavaScript module for distributed tracing.
+- Large refactoring of core components to simplify the code base, improve maintainability, and increase test coverage.
+- Bug fixes, UX improvements, and maintenance.
+
+Keep reading for the details.
+
+## Breaking changes
+
+- [#2807](https://github.com/grafana/k6/pull/2807) Use non-zero exit codes for tests aborted by Ctrl+C or the REST API.
+
+  Aborting a test run with Ctrl+C will now exit with code `105`, and stopping via the REST API will exit with code `103`.
+
+- [#2903](https://github.com/grafana/k6/pull/2903) JS modules are now cached by absolute, and not relative paths.
+
+  This fixes a regression introduced in v0.42.0 ([#2773](https://github.com/grafana/k6/pull/2773)), but has the side-effect of changing how modules are cached internally, which may cause issues for scripts that relied on the previous behavior.
+
+
+## New Features
+
+### xk6-browser is now a built-in module [#2884](https://github.com/grafana/k6/pull/2884)
+
+[xk6-browser](https://github.com/grafana/xk6-browser), a k6 extension that enables browser automation and end-to-end testing from k6 scripts, now comes bundled with k6 under the `k6/experimental/browser` import path. This makes it more accessible, as users no longer need to build their own binary with [xk6](https://github.com/grafana/xk6), or use a separate binary to run browser tests.
+
+All current xk6-browser scripts that work with v0.8.0 will continue to work with the built-in module in k6 v0.43.0, by simply changing the import path from `k6/x/browser` to `k6/experimental/browser`, and setting the environment variable `K6_BROWSER_ENABLED` to `true`. You can see a script example [here](https://github.com/grafana/k6/blob/v0.43.0/samples/experimental/browser.js), and read the updated [xk6-browser documentation](https://k6.io/docs/javascript-api/xk6-browser/).
+
+The module is currently under the `experimental` namespace as we reserve the decision to introduce breaking changes in the future, but our mid-term goal is to drop the `experimental` label and make browser support a stable part of the k6 feature set.
+
+
+### Native support for JavaScript's `async`/`await` [#2830](https://github.com/grafana/k6/pull/2830)
+
+In [v0.35.0](https://github.com/grafana/k6/releases/tag/v0.35.0) we added support for asynchronous functionality in k6 scripts with the addition of `Promise`.
+
+This was very useful, but the experience wasn't very friendly, as users had to use the `.then()` API to resolve Promises, instead of the `async`/`await` syntax available in most other JavaScript runtimes since ECMAScript 2016. [Some workarounds were possible](https://github.com/grafana/k6/issues/779#issuecomment-964027280), but it required a separate build pipeline to transpile the syntax into the older ES5.1+ standard supported by k6.
```

Comment:
```suggestion
This was very useful, but the experience wasn't very friendly, as users had to use the `.then()` API to chain Promises, instead of the `async`/`await` syntax available in most other JavaScript runtimes since ECMAScript 2016. [Some workarounds were possible](https://github.com/grafana/k6/issues/779#issuecomment-964027280), but it required a separate build pipeline to transpile the syntax into the older ES5.1+ standard supported by k6.
```

Reply:
Although I am not certain that you need all of this intro into the topic, but will leave the others to comment more on that. 

I kind of prefer to keep the release notes shorter so people can actually read them instead of seeing that they are 3 pages and skimming over the headers. And in order for that to happen I feel like we should not add too much information. In this particular case - if people don't know what async/await is used for .... uh I don't think it will help them. Linking to the workaround is also not helpful or useful for anyone who hasn't used it. And the people who did - should not need to be linked to it in order to know that they no longer need it.

Reply:
I also prefer short release notes, but at the same time, we're highlighting a popular feature we were missing up until now, so I think it makes sense to give some background information about our previous shortcomings. This is not explaining what `async`/`await` is for, but mentioning what the UX was before it was added.

I'd rather leave this as is, but if there are more votes for removing it, I'll change it.

Reply:
Okay - let's leave it, but can we merge the proposed change ?

Reply:
@MattDodsonEnglish I applied your suggestion with minor changes in 2acbcbc9. Let me know.